### ISSUE TITLE
Axis max / min are not allowed for date axis

### ISSUE THREAD

**Issue by teledemic** [n3-charts/line-chart#274]
The sanitizeExtrema function will remove an axis min and/or max if it's a date. This works fine if you skip sanitizeExtrema.
I would submit a pull request but never use coffeescript. I have been adding this to my sanitizeExtrema js:
```
if (options.min instanceof Date && !isNaN(options.min.valueOf()) && options.max instanceof Date && !isNaN(options.max.valueOf())) {
    return true;
}
```
Thanks!

**Comment by chaosmail** [comment]
Hi @teledemic 
thanks for reporting this, it is indeed an annoying bug! We will try to fix this as soon as possible - but feel free to submit a PR. Coffeescript is easy to get started; especially if you have to just add this little `Date` check and a small unit test. Let me know if you need guidance on this.
Best,
Christoph

**Comment by chaosmail** [comment]
@MattiSG fixed this in #286
